tests/test_interface.py

```diff
@@ -47,7 +49,7 @@ def test_duplicate_component_iw2d_import():
         pytest.skip("IW2D is not installed")
```

Comment:
these four lines are repeated over and over. I would create a function once for all and call it where needed.

or even better: do this once at the beginning and use `@pytest.mark.skipif` before each test to skip

Reply:
Done!

Reply:
:+1: